package/doc/sphinx/source/documentation_pages/lib/picklable_file_io.rst

```diff
@@ -0,0 +1 @@
+.. automodule:: MDAnalysis.lib.picklable_file_io
```

Comment:
Did you build the docs and all functions/classes show up?

Reply:
did they?

Reply:
Yes.